lib/tpm2.h

```diff
@@ -340,4 +340,6 @@ tool_rc tpm2_policy_nv_written(ESYS_CONTEXT *esys_context,
         ESYS_TR policy_session, ESYS_TR shandle1, ESYS_TR shandle2,
         ESYS_TR shandle3, TPMI_YES_NO written_set);
 
+uint32_t fix_esys_hierarchy(TPMI_RH_HIERARCHY hierarchy);
```

Comment:
@tstruk is this going to be an errata? if so, should this function be renamed as such with an errata number?

Reply:
My understanding is that the errata mechanism is targeted for TPM spec issues that are fixed by errata. The ESAPI issue that this change fixes is not TPM spec related, but it is the implementation issue, so no errata number exists for it.

Reply:
> My understanding is that the errata mechanism is targeted for TPM spec issues that are fixed by errata. The ESAPI issue that this change fixes is not TPM spec related, but it is the implementation issue, so no errata number exists for it.

I see. LGTM.

Reply:
@tstruk, I think we should leave a comment explaining the necessity for this function and reference the tpm2-tss issue#

Reply:
Agree. Will send an update soon.